connector/pom.xml

```diff
@@ -388,14 +479,40 @@
                     <artifactId>unused</artifactId>
                     <groupId>org.spark-project.spark</groupId>
                 </exclusion>
+                <exclusion>
+                    <groupId>com.fasterxml.jackson.datatype</groupId>
+                    <artifactId>jackson-datatype-jdk8</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>com.fasterxml.jackson.datatype</groupId>
+                    <artifactId>jackson-datatype-jsr310</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>com.fasterxml.jackson.datatype</groupId>
+                    <artifactId>jackson-datatype-guava</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>com.fasterxml.jackson.datatype</groupId>
+                    <artifactId>jackson-datatype-joda</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>com.fasterxml.jackson.module</groupId>
+                    <artifactId>jackson-module-parameter-names</artifactId>
+                </exclusion>
             </exclusions>
+            <scope>provided</scope>
         </dependency>
 
         <dependency>
             <groupId>com.huawei.boostkit</groupId>
             <artifactId>boostkit-omnidata-core</artifactId>
-            <version>1.0.0-aarch64</version>
+            <version>1.0.0</version>
+            <classifier>${dep.os.arch}</classifier>
             <exclusions>
+                <exclusion>
+                    <groupId>org.bouncycastle</groupId>
+                    <artifactId>bcprov-jdk15on</artifactId>
+                </exclusion>
```

Comment:
we shouldn't exclusion bcprov-jdk15on, otherwise it will throw exception in openlookeng when ssl enabled.
![image](https://user-images.githubusercontent.com/7078005/132081475-1bf47b40-aa4f-44ff-a480-dd28d87b97b1.png)